### PR TITLE
Show list of twiddles

### PR DESCRIPTION
**Pull request by Gaurav0**
Creates a /twiddles route that shows a list of the user's twiddles.

Issue #93

### CODE REVIEW

**Comment by Gaurav0** [comment]
@joostdevries @stefanpenner @rwjblue PR is complete. Please review.

File: app/components/saved-twiddles-row.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+export default Ember.Component.extend({
+  tagName: 'tr',
+
+  numFiles: Ember.computed('gist.files.[]', function() {
+    return this.get('gist.files.length');
+  }),
+
+  filesTitle: Ember.computed('gist.files.@each.filePath', function() {
+    return this.get('gist.files').toArray().map((file) => {
```
Comment (stefanpenner) [inline app/components/saved-twiddles-row.js]: `mapBy('filePath')` would also work

File: app/serializers/gist.js [hunk 2]
```diff
@@ -1,24 +1,45 @@
 import ApplicationSerializer from './application';
 
 export default ApplicationSerializer.extend({
+  seq: 0,
```
Comment (stefanpenner) [inline app/serializers/gist.js]: should be set in init

File: app/twiddles/route.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  beforeModel() {
+    return this.session.fetch('github-oauth2').catch(function(error) {
```
Comment (stefanpenner) [inline app/twiddles/route.js]: =>

**Pull request merged**